data/json/items/tool/misc.json

```diff
@@ -707,14 +707,14 @@
     "id": "teleumbrella",
     "type": "ARMOR",
     "category": "tools",
-    "name": { "str": "telescoping umbrella" },
-    "description": "A telescoping umbrella which collapses down for easy storage, useful for keeping dry when wielded.",
-    "weight": "118 g",
-    "volume": "500 ml",
+    "name": { "str": "telescoping umbrella (retracted)", "str_pl": "telescoping umbrellas (retracted)" },
```

Comment:
I find it a bit odd that we're using both retracted and collapsed. Would make more sense to just use collapsed throughout

Reply:
yea, that makes sense, ill change it